src/site/content/en/accessible/control-focus-with-tabindex/index.md

```diff
@@ -133,9 +133,9 @@ method on it.
 
 ```html/2-3
 <div role="toolbar">
-  <button tabindex="-1">Undo</div>
-  <button tabindex="-1">Redo</div>
-  <button tabindex="0">Cut</div>
+  <button tabindex="-1">Undo</button>
```

Comment:
There is one more at line number 126,127,128.

Reply:
Oh, I must have overlooked it.